library/src/main/java/com/novoda/downloadmanager/DownloadManagerBuilder.java

```diff
@@ -60,12 +59,12 @@
     private TimeUnit timeUnit;
     private long frequency;
     private Optional<LogHandle> logHandle;
-    private StorageRequirementsRule storageRequirementsRule;
+    private StorageRequirementRules storageRequirementRules;
 
     public static DownloadManagerBuilder newInstance(Context context, Handler callbackHandler, @DrawableRes final int notificationIcon) {
         Context applicationContext = context.getApplicationContext();
 
-        StorageRequirementsRule storageRequirementsRule = StorageRequirementsRule.withPercentageOfStorageRemaining(TEN_PERCENT);
+        StorageRequirementRules storageRequirementRule = StorageRequirementRules.newInstance();
```

Comment:
We do not add a rule by default, just create the empty list. It is the responsibility of the client to add a rule if they feel one is necessary.